src/test/kotlin/no/nav/familie/oppdrag/rest/OppdragControllerIntegrationTest.kt

```diff
@@ -0,0 +1,62 @@
+package no.nav.familie.oppdrag.rest
+
+import no.nav.familie.kontrakter.felles.oppdrag.Opphør
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsperiode
+import no.nav.familie.oppdrag.iverksetting.OppdragMapper
+import no.nav.familie.oppdrag.service.OppdragService
+import org.junit.Ignore
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+import org.springframework.jms.annotation.EnableJms
+import org.springframework.test.context.ActiveProfiles
+import java.math.BigDecimal
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Configuration
+@ComponentScan("no.nav.familie.oppdrag") class TestConfig
+
+@ActiveProfiles("dev")
+@SpringBootTest(classes = [TestConfig::class],properties = ["spring.cloud.vault.enabled=false"])
+@EnableJms
+internal class OppdragControllerIntegrasjonTest {
+
+    val localDateTimeNow = LocalDateTime.now()
+    val localDateNow = LocalDate.now()
+
+    val utbetalingsoppdrag = Utbetalingsoppdrag(
+            Utbetalingsoppdrag.KodeEndring.NY,
+            "TEST",
+            "SAKSNR",
+            "FNR-014",
+            "SAKSBEHANDLERID",
+            localDateTimeNow,
+            listOf(Utbetalingsperiode(false,
+                                      Opphør(localDateNow),
+                                      localDateNow,
+                                      "KLASSE A",
+                                      localDateNow,
+                                      localDateNow,
+                                      BigDecimal.ONE,
+                                      Utbetalingsperiode.SatsType.MND,
+                                      "UTEBETALES_TIL",
+                                      1))
+    )
+
+    @Autowired lateinit var oppdragService : OppdragService
+
+    @Ignore("Krever at MQ og PostgresSql kjører lokalt. Se readme")
```

Comment:
Veldig bra med test 🙌  Vi kan alternativt legge på en `@DisabledIfEnvironmentVariable(named = "CIRCLECI", matches = "true")`, så kjører den lokalt by default. Regner med at vi alle kjører opp MQ og postgres lokalt når vi jobber i familie-oppdrag uansett :)

Reply:
Ja, det skal jeg gjøre. Denne testen er mer en main-metode. For å teste at tx funker, så må jeg få injected exceptions inn i transaksjonene. Tar gjerne litt tips på hvordan jeg skal få til det